NotificationHubs/src/com/windowsazure/messaging/RetryUtil.java

```diff
@@ -21,6 +27,8 @@
     private RetryUtil() {
     }
 
+    private static int[] retriableStatusCodes = new int[] {500, 503, 504, 408};
```

Comment:
Unless Retry-After changes the story, I believe 429/403 should be included in this as well. (403 for frustrating legacy reasons)